auction-app-frontend/src/services/ProductService.js

```diff
@@ -0,0 +1,16 @@
+import api from 'services/Api';
+
+function getNewArrivalsProducts(page, size) {
+    return api.get(`/products?page=${page}&size=${size}&sortKey=startDate&sortDirection=desc`);
+}
+
+function getLastChanceProducts(page, size) {
+    return api.get(`/products?page=${page}&size=${size}&sortKey=endDate&sortDirection=asc`);
+}
```

Comment:
Instead of these two functions, we can create one - `getProducts(page, size, sortKey, sortDirection)`.